cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="staff_only">
+                <%- gettext('Hide from students') %>
```

Comment:
All labels should refer to "learners" rather than "students": 
"Hide from students" -> "Hide from learners"